README.md

```diff
@@ -1,2 +1,69 @@
 # fastify-get-head
-Fastify plugin to set a HEAD route handler for each GET handler previously registered
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/fastify/fastify-cookie.svg)](https://greenkeeper.io/)
```

Comment:
remove greenkeeper badge, it's out of date and greenkeeper is not a thing anymore